terraformer_cloud_init.tf

```diff
@@ -92,18 +92,18 @@ data "cloudinit_config" "terraformer_cloud_init_tasks" {
   #   parameter containing the VNC user's username
   part {
     content = templatefile(
-      "${path.module}/cloud-init/write-aws-config.tpl.sh", {
+      "${path.module}/cloud-init/write-terraformer-aws-config.tpl.sh", {
         assessor_account_role_arn                     = var.assessor_account_role_arn
         aws_region                                    = var.aws_region
         permissions                                   = "0400"
         read_cool_assessment_terraform_state_role_arn = module.read_terraform_state.role.arn
         organization_read_role_arn                    = data.terraform_remote_state.master.outputs.organizationsreadonly_role.arn
         terraformer_role_arn                          = aws_iam_role.terraformer_role.arn
-        vnc_read_parameter_store_role_arn             = aws_iam_role.gucamole_parameterstorereadonly_role.arn
+        vnc_read_parameter_store_role_arn             = aws_iam_role.guacamole_parameterstorereadonly_role.arn
```

Comment:
I believe my suggestions to rework the other templates applies here. I'm not sure if you want to break that into another PR if this is true but I wanted to at least mention it since the file's being touched.

Reply:
Please see commit d9b1e38.